source/help/messaging/attaching-files.rst

```diff
@@ -0,0 +1,74 @@
+Attaching Files
```

Comment:
The index file for this guide section needs to be updated when the file name changes:

https://github.com/mattermost/docs/blob/master/source/guides/user.rst